modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,112 @@
+
+[[adding-api-instances-to-groups]]
+== Add API Instances to Existing API Groups
+
+You might want to add API instances to an existing API Group to widen the business problem it solves. 
+
+To add API instances to an existing API Group:
+
+. From the left menu of API Manager, click *API Groups*.
++
+.The screenshot illustrates the page that is displayed for API Groups.
+image:add-api-instance-modify-api-groups.png[Add API Instances to API Groups]
++
+<1> 
+<2> 
+. On the API Groups details page, expand the API Group to view its available versions.
+. Click the version of the API Group to which you want to add the API instance and click *Add Instance*.
++
+The *Creating a new group instance* page is displayed.
+. Click *Add*.
+. If the API instance and the API Group belong in the same business group or organization, select the environment, API instance, and version that you want to add to the API Group.
+. If the API instance and the API Group belong in different business groups:
+.. Click *Add Organization*.
+.. Select the organization or business group in which your API instance is available.
+.. From the environment, select the API instance, and version that you want to add.
+. Click *Save*.
++
+
+
+To publish your API Group in Exchange, see xref:api-groups-publishing-to-exchange.adoc[publishing API Groups to Exchange].
+
+[[removing-api-instances-from-groups]]
+== Remove API Instances from API Groups
+
+If your business requirement changes, you might want to modify an API Group by removing an existing API instance.
+
+To remove an API instance from an API Group:
+
+. From the left menu in API Manager, click *API Groups*.
+. On the API Groups details page, expand the API Group to view its available versions.
+. Click the version of the API Group to view the instance.
++
+The API Group instance page is displayed.
++
+image:remove-api-instance-api-groups.png[Remove API Instances from API Groups]
+. From the top-right corner, click *Edit*.
+. Click the cross icon next to the API instance that you want to remove. 
+. If you want to remove all the API instances from a specific organization or business group, click the cross icon next to the organization name.
++
+Note that if the API Instance you removed is the last API linked to an asset, you must create a new version of the API Group.
+[PA: I don't see this option in my version of the product.]
+. Click OK.
+
+[[creating-new-versions-of-groups]]
+== Create new versions of API Groups
+If you modify an API Group to suit a changing business requirement, you might need to version it appropriately.
+
+// add in what scenario would the users need to create new versions of API Groups.
+
+To create additional versions of an API Group:
+
+. From the left menu In API Manager, click *API Groups*. 
++
+image::api-products-create-new-version.png[Creating New Version of API Group]
```

Comment:
same, please use API groups

Reply:
It's not a high priority to change internal file names at this point.